app/src/main/java/com/internship/move/feature/onboarding/adapter/ViewPagerAdapter.kt

```diff
@@ -0,0 +1,20 @@
+package com.internship.move.feature.onboarding.adapter
+
+import androidx.fragment.app.Fragment
+import androidx.viewpager2.adapter.FragmentStateAdapter
+import com.internship.move.feature.onboarding.model.ViewPagerItem
+
+class ViewPagerAdapter(private val currentList : ArrayList<ViewPagerItem>, fragment : Fragment) : FragmentStateAdapter(fragment) {
+
+    override fun createFragment(position: Int): Fragment {
```

Comment:
I prefer inlining such short functions
`override fun createFragment(position: Int): Fragment = ViewPagerFragment(currentList[position])`
similarly for `getItemCount`

But it's a subjective choice tbh, up to you